### PR TITLE
Add calico-node -init command to mount bpffs.

### PR DESCRIPTION
**Pull request by fasaxc**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: pkg/nodeinit/calico-init_windows.go
Comment (neiljerram) [inline pkg/nodeinit/calico-init_windows.go:17]: Is this needed on Windows?

File: pkg/nodeinit/calico-init_linux.go
Comment (neiljerram) [inline pkg/nodeinit/calico-init_linux.go:40]: Does 2 have a special meaning here?  If so, maybe worth a comment to explain that.

**Review by neiljerram — APPROVED** [review]
LGTM, just a couple of nits

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Comment by fasaxc** [comment]
Tests passed but the result isn't showing up for some reason

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
